access/types/forms.py

```diff
@@ -689,11 +706,14 @@ def grade_text(self, configuration, value, hints=None):
             method = "regexp"
         elif "correct" in configuration:
             accept = configuration["correct"]
-        if not accept is None:
+            if method == 'subdiff':
```

Comment:
Can `method` contain modifiers such as `ignorews` here? Then `method == 'subdiff'` would not work.